pages/api/rollup/[cid].js

```diff
@@ -0,0 +1,24 @@
+import { retrieveFilesData } from '../../../util/web3Storage'
```

Comment:
I have a different path here since we would be planning to get the skillId, but I can branch off this one later.

Reply:
Nevermind, we will be doing community based rollups.